flow-api/src/main/java/com/sequenceiq/flow/api/FlowEndpoint.java

```diff
@@ -80,4 +87,23 @@ public interface FlowEndpoint {
     @ApiOperation(value = "Check if there is a running flow for flow id", produces = "application/json", notes = "Flow log operations",
             nickname = "hasFlowRunningByFlowId")
     FlowCheckResponse hasFlowRunningByFlowId(@PathParam("flowId") String flowId);
+
+    @GET
+    @Path("/logs/flowIds")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Get flow logs by a list of flow ids - Input size max 50", produces = "application/json", notes = "Flow log operations",
+            nickname = "getFlowLogStatusesByFlowIds")
```

Comment:
please change the nickname too to `getFlowLogsByFlowIds`

Reply:
done